insecure_generate_tls.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/sh
 
+# you can set the hostname if you want, but it'll default to localhost
+if [ -z "$CERT_HOSTNAME" ]; then
+    CERT_HOSTNAME="localhost"
+fi
 
-KANI_TMP=/tmp/kanidm/
+# also where the files are stored
+if [ -z "$KANI_TMP" ]; then
```

Comment:
At what point do you convert this script to python .....

Reply:
sooooon... or a rust build target that makes and deploys the certs 🤔